docs/book/features/file_volume.md

```diff
@@ -15,7 +15,9 @@ In order to utilize this feature in your vSphere environment, you need to make s
 
 - Establish a dedicated file share network connecting all the kubernetes nodes and make sure this network is routable to the vSAN File Share network.  Refer to [Network Access of vSAN File Share](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-EFC00FFF-E720-44F1-B229-4C13687E6B85.html) to understand the setup better.
 
-- Configure the kubernetes nodes with the same DNS server as the one used to configure the file services in the vSAN cluster configuration. This will help the nodes resolve the access points it gets from the File service and mount the file share as a volume while creating a pod. Refer to [vSAN - Configure File Service](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vsan.doc/GUID-CA9CF043-9434-454E-86E7-DCA9AD9B0C09.html) for more details.
+- Configure the kubernetes nodes with the same DNS server as the one used to configure the file services in the vSAN cluster configuration. This will help the nodes resolve the access points it gets from the File service and mount the file share as a volume while creating a pod. In order to retrieve the file service FQDN, head over to the `Configure` tab of your vSAN cluster and navigate to the File Service section as shown in the diagram below. At the end of this step, you should be able to resolve the File service FQDN on each of your kubernetes nodes.
```

Comment:
we can use multiple DNS servers, so it should not be the case that we need to use only file service dns.

Reply:
Changed it